### ISSUE TITLE
Support for Amazon Web Services

### ISSUE THREAD

**Issue by alevere** [oasis-tcs/openc2-apsc-stateless-packet-filter#140]
AWS released a new network-based firewall service aptly named 'AWS Network Firewall' that supports both stateless and stateful packet filtering through it. See https://docs.aws.amazon.com/network-firewall/latest/APIReference/API_CreateRuleGroup.html

The second part of supporting AWS would be supporting Security Groups. 
The last part to fully support AWS would be to support Network ACLs.

**Comment by Vasileios-Mavroeidis** [comment]
Yes, i noticed a couple weeks ago. I have extracted the functionality that we need. Im going to upload it when i have some time and before our next AP meeting.

-Vasileios

On Dec 3, 2020, at 7:23 PM, alevere <notifications@github.com<mailto:notifications@github.com>> wrote:



AWS released a new network-based firewall service aptly named 'AWS Network Firewall' that supports both stateless and stateful packet filtering through it. See https://docs.aws.amazon.com/network-firewall/latest/APIReference/API_CreateRuleGroup.html

The second part of supporting AWS would be supporting Security Groups.
The last part to fully support AWS would be to support Network ACLs.
